### DISCUSSION TITLE
WSL2 run Docker (Docker Desktop) inside Fedoraremix?

### DISCUSSION THREAD

**Discussion by mattgagliardi** [WhitewaterFoundry/Fedora-Remix-for-WSL#206] [Q&A] [answered]
Long/short I'm trying to execute docker commands inside Fedoraremix via Docker Desktop's integration.  This works fine in my Ubuntu distro (on WSL2).  Inside Fedoraremix I can run commands like "docker --version" or "docker --help" and get the expected output, but when I attempt "docker run <whatever>" it all goes belly-up and I receive the following error message:

`docker: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?.`

I'm mystified at this point.  Docker Desktop is fully upgraded, as is WSL2 (and as I said, this works OK in my Ubuntu distro).  I've looked around for directions specific to FR but haven't been able to find any.  Does anyone have experience getting this to work?  FWIW I'm running v38.  TIA!

**Comment by crramirez** [comment]
Looks like the integration with WSL2 in Docker Desktop 4.25 is broken. WSL
Integration doesn't work and causes WSL to hang · Issue #13774 ·
docker/for-win (github.com)
<https://github.com/docker/for-win/issues/13774> Have
you tested Ubuntu with this version?

**Reply by mattgagliardi** [reply]
Yes, the versions mentioned in the link you posted are what I'm currently using, however I'm not experiencing any issues in Ubuntu...only in Fedoraremix.

**Reply by mattgagliardi** [reply, marked as answer]
The Docker build at this [link](https://github.com/docker/for-win/issues/13764#issuecomment-1798242872) appears to have resolved the issue in my FR deployment.
